{{cookiecutter.repository_name}}/debug/evaluate.py

```diff
@@ -0,0 +1,5 @@
+from guild.commands.run import run
+
+
+if __name__ == "__main__":
+    run(["evaluate", "-y", "n_workers=0", "--debug-sourcecode=."])
```

Comment:
Imo we should labels these by default :) E.g. `debug-evaluate`/`debug-train`

Reply:
What do you mean? Do you want to rename the file?

Reply:
Nevermind you mean add labels. I tried it but it didn't work for me. Do you know how to?

Reply:
`--label debug-train` works for me

Reply:
`-l` should work too but didn't seem to work with some version of guild for some reason